ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -545,18 +546,27 @@ aaa_server_groups:
 ```yaml
 aaa_authentication:
   login:
-    default: < group | local | none >
-    serial_console: < group | local | none >
+    default: < group name >  | local | none >
+    serial_console: < group name >  | local | none >
```

Comment:
It is still confusing, since we need the "group" word to be there. Also there is a misplaced `>`
I suggest:
`default: < group group_name | local | none > < group group_name | local | none > ...`

Reply:
Good point, fixed!